.gitignore

```diff
@@ -1,36 +1,43 @@
+# Setup
+/seed-database/
+*.env
+
+# Python
+/venv/
+/ve/
 *.pyc
-*~
-.vagrant
+
+# OS
+.DS_Store
+
+# Editors
+/.vscode/
+/.idea/
```

Comment:
I used to be against OS or editor specific gitignore patterns, and was recommending using [global gitignore for that](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) but I changed my mind.

Reasons:
- Setting up global gitignore is added friction, especially for new team members or external contributors.
- Automated tools (like jscpd, eslint, prettier) and some IDEs do not look into global gitignore file